api/cpp/docs/cmake.md

```diff
@@ -42,7 +42,7 @@ First you need to install the prerequisites:
 * Install Rust by following the [Rust Getting Started Guide](https://www.rust-lang.org/learn/get-started). If you already
   have Rust installed, make sure that it's at least version 1.60 or newer. You can check which version you have installed
   by running `rustc --version`. Once this is done, you should have the ```rustc``` compiler and the ```cargo``` build system installed in your path.
-* **[cmake](https://cmake.org/download/)** (3.21 or newer)
+* **[cmake](https://cmake.org/download/)** (3.23 or newer)
```

Comment:
Don't we have to also bump the `cmake_minimum_required(VERSION ...)` calls in our CMakeLists.txt (and consequently in the slint-cpp-template)? The new `INITIALIZE_FROM_VARIABLE` will be visible to every user after all, since it's in `SlintMacro.cmake`.

A more complex alternative would be to implement the inheritance inside a gigantic generator expression, but TBH, I prefer your approach very much and I for one don't mind a version bump.